decidim-core/app/assets/javascripts/decidim/account_form.js.es6

```diff
@@ -0,0 +1,27 @@
+/**
+ * Since the delete account has a modal to confirm it we need to copy the content of the
+ * reason field to the hidden field in the form inside the modal.
```

Comment:
No, it isn't. The problem is the foundation reveal is rendered outside the real form and it is also a copy of the template marked as a popup. I know it's not a perfect solution but I needed this to solve the problem.